### DISCUSSION TITLE
The Kubb loses the path to the files if $ref is used

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [kubb-labs/kubb#859] [Q&A] [answered]
Hi there. I'm getting an error when trying to generate types from a schema, where the paths and responses use $ref in files with their declarations for reuse. What am I doing wrong?

![image](https://github.com/kubb-project/kubb/assets/19804652/0b8ad2be-e516-42ad-805a-e9b6c3ba3d71)
![image](https://github.com/kubb-project/kubb/assets/19804652/c852970e-1531-45f1-ac9d-47177b5b02cb)
<img width="282" alt="Снимок экрана 2024-03-11 в 19 14 14" src="https://github.com/kubb-project/kubb/assets/19804652/b9ed2227-f939-44d3-9345-9a7c002d84a2">

**Comment by stijnvanhulle** [comment]
Right now we don't have support for `ref` outside the main Swagger/OpenAPI file. Something we should plan as an improvement

**Comment by stijnvanhulle** [comment, marked as answer]
@popuguytheparrot Version [2.14.0](https://www.npmjs.com/package/@kubb/core/v/2.14.0) of Kubb will resolve this issue.
